### DISCUSSION TITLE
Test new Rule doens't work

### DISCUSSION THREAD

**Discussion by fedex03** [firefly-iii/firefly-iii#7239] [Firefly III]
Hi there,

I updated FireFly to the latest version this moring.

I found a different behaviour when I add a new rule and I push the button "See correspondent transaction".
With the previous version, I see the transactions that match the rule. 
Now the list is empty, but if I test the rule in the rule page, with the "becher" icon, the transactions are shown.

![immagine](https://user-images.githubusercontent.com/16340343/225323671-b7efa9f7-29bd-4591-84c0-86e0e221d2ef.png)

@JC5 : Thank you very much for your efforts!!!

**Comment by JC5** [comment]
Yes I'm aware, thanks for letting me know!

https://github.com/firefly-iii/firefly-iii/issues/7221

**Reply by fedex03** [reply]
Your welcome.
Thanks
